tests/run-make/prefer-rlib/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// Check that `foo.rs` prefers to link to `bar` statically, and can be executed even if the `bar`
+// library artifacts are removed.
+
+//@ ignore-cross-compile
+
+use run_make_support::{dynamic_lib_name, path, run, rust_lib_name, rustc};
+use std::fs::remove_file;
+
+fn main() {
+    rustc().input("bar.rs").crate_type("dylib").crate_type("rlib").run();
+    assert!(path(rust_lib_name("bar")).exists());
+    rustc().input("foo.rs").run();
+    remove_file(rust_lib_name("bar")).unwrap();
```

Comment:
```suggestion
    fs_wrapper::remove_file(rust_lib_name("bar"));
```
+ `use run_make_support::{dynamic_lib_name, fs_wrapper, ...}`.

We're trying to use all `fs` operations from the support library, to provide better error reporting and to make sure that we react to all errors. The `fs_wrapper` PR has been merged, so now ideally all `fs` operations should be done through it.

Reply:
Ah nice! Did it as well.